.github/workflows/test.yaml

```diff
@@ -2,6 +2,8 @@ name: Tests
 
 on:
   push:
+    branches:
+      - 'main'
   pull_request:
```

Comment:
Teraz nám pri každom commite v PR bežia testy dvakrát. To nie je dobré. S touto úpravou budú testy bažeť raz pri každom pushi v PR a raz pri push (čiže merge) do mainu.